<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QLabWorkerWgt</class>
 <widget class="QWidget" name="QLabWorkerWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabTabWgt" name="tabWidget">
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="tabLabChose">
      <attribute name="title">
       <string>Выбор варианта</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabVarChoseWgt" name="wgtLabChose" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabStatusTable">
      <attribute name="title">
       <string>Кодировка состояния</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabStatusTableWgt" name="wgtStatusTable" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTransitionTable">
      <attribute name="title">
       <string>Таблица переходов</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabTransitionTableWgt" name="wgtTransitionTable" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabStructureTable">
      <attribute name="title">
       <string>Структурная таблица</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabStructureTableWgt" name="wgtStructureTable" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCarnoCard3">
      <attribute name="title">
       <string>Минимизация</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Tab 1</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabReport">
      <attribute name="title">
       <string>Отчет</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabReportWgt" name="wgtReport" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonBack">
       <property name="text">
        <string>&lt;- &amp;Назад</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="text">
        <string>&amp;Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonNext">
       <property name="text">
        <string>&amp;Далее -&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLabVarChoseWgt</class>
   <extends>QWidget</extends>
   <header>qlabvarchosewgt.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QLabStatusTableWgt</class>
   <extends>QWidget</extends>
   <header>qlabstatustablewgt.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QLabTabWgt</class>
   <extends>QTabWidget</extends>
   <header>qlabworkerwgt.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QLabReportWgt</class>
   <extends>QWidget</extends>
   <header>qlabreportwgt.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QLabTransitionTableWgt</class>
   <extends>QWidget</extends>
   <header>qlabtransitiontablewgt.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QLabStructureTableWgt</class>
   <extends>QWidget</extends>
   <header>qlabstructuretablewgt.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
